changedetectionio/notification.py

```diff
@@ -29,6 +32,7 @@
 
 def process_notification(n_object, datastore):
 
+    logging.basicConfig(stream=sys.stderr, level=logging.DEBUG)
```

Comment:
we should probably do this across the whole app i think, unsure

Reply:
oh sorry, I didn't want that to be checked in, just used it for debugging the failing test.